### PR TITLE
Corrected ambiguity in ZSTR_TEMP referring to both TEMP1_ID and TEMP2…

### PR DESCRIPTION
**Pull request by ptsneves**
Answer to Issue #2953 
This ambiguity causes problems with the scripts' getValue().

A simple rename is unadvised because the user doesn't know which of the "discovered" sensors represents the intended data.

Also there is an inconsistency in frsky_sport.cpp where sport sensors for ZSTR_TMP

is valid for the range T1_FIRST_ID, T2_LAST_ID when the T1_FIRST_ID, T1_LAST_ID and T1_FIRST_ID, T1_LAST_ID already exist.

I have not created a pull request because this patch is very small and applicable to 2.1.3 tag and the master and next branch are already too far ahead and with merge conflicts in frsky_d.cpp.

If anybody can guide me where the changes of frsky_d.cpp are applicable in the next or master branch let me know and I will make a pull request

### CODE REVIEW

File: radio/src/telemetry/frsky_sport.cpp [hunk 1]
```diff
@@ -64,7 +64,8 @@ const FrSkySportSensor sportSensors[] = {
   { A3_FIRST_ID, A3_LAST_ID, ZSTR_A3, UNIT_VOLTS, 2 },
   { A4_FIRST_ID, A4_LAST_ID, ZSTR_A4, UNIT_VOLTS, 2 },  
   { BATT_ID, BATT_ID, ZSTR_BATT, UNIT_VOLTS, 1 },
-  { T1_FIRST_ID, T2_LAST_ID, ZSTR_TEMP, UNIT_CELSIUS, 0 },
+  { T1_FIRST_ID, T1_LAST_ID, ZSTR_TEMP1, UNIT_CELSIUS, 0 },
+	{ T2_FIRST_ID, T2_LAST_ID, ZSTR_TEMP2, UNIT_CELSIUS, 0 },
```
Comment (projectkk2glider) [inline radio/src/telemetry/frsky_sport.cpp]: Indentation problem.

**Comment by projectkk2glider** [comment]
Looks good. Do not worry about the `next` branch for now.

**Comment by bsongis** [comment]
I am ok as well. It's consistent with A1, A2 etc. I don't like Tmp1 but no choice, we only have 4 chars available

**Comment by ptsneves** [comment]
The Tmp was due to the 4 character limitations indeed. There are other references to Tmp in the general code so it is not terribly inconsistent. 

In Monday I will correct the indentation problem, if necessary.

**Comment by ptsneves** [comment]
The Indentation problem is corrected.

**Comment by bsongis** [comment]
Thanks!

**Pull request merged**